<launch>

  <node name="track_classifier" pkg="rospix_classification" type="batch.py" output="screen">

    <rosparam file="$(find rospix_classification)/config/batch_processing.yaml" />

  </node>

</launch>
